modules/ecs-service/remote-state.tf

```
@@ -30,47 +44,113 @@ locals {
   public_domain_no_service_name  = format("%s-%s.%s.%s", "public", local.cluster_type, var.environment, local.zone_domain)
   private_domain_no_service_name = format("%s-%s.%s.%s", "private", local.cluster_type, var.environment, local.zone_domain)
 
-  # tenant to domain mapping
-  vanity_domain_names = {
-    tenantexample = {
-      "dev"     = "example-dev.com",
-      "staging" = "example-staging.com",
-      "prod"    = "example-prod.com",
-    },
-  }
+  vanity_domain_zone_id = one(data.aws_route53_zone.selected_vanity[*].zone_id)
+
+  unauthenticated_paths = local.is_nlb ? ["/"] : var.unauthenticated_paths
+
+  # NOTE: this is the rare _not_ in the ternary purely for readability
+  full_urls = !local.use_lb ? [] : [for path in local.unauthenticated_paths : format("%s://%s%s", local.http_protocol, local.lb_fqdn, trimsuffix(trimsuffix(path, "*"), "/"))]
 
-  vanity_domain         = local.vanity_domain_names[var.tenant][var.stage]
-  vanity_domain_zone_id = try(one(data.aws_route53_zone.selected_vanity[*].zone_id), null)
 }
 
 module "vpc" {
   source  = "cloudposse/stack-config/yaml//modules/remote-state"
-  version = "1.4.1"
+  version = "1.4.3"
```

change all remote state modules to `1.5.0`
```suggestion
  version = "1.5.0"
```